modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/resources/META-INF/resources/js/components/Error.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayAlert from '@clayui/alert';
+import React from 'react';
+
+export default function Error() {
```

Comment:
@ramonmsp maybe we could use a more descriptive name, like ErrorFeedback or something similar. What do you think? Other possibility would be to not create a separate component for this. Given its just a simple ClayAlert, you could just call it directly in the return.

Reply:
@naraandrade I think it's better keep the external component because of div that wrap ClayAlert. I'll change the name :D